CHANGELOG.md

```diff
@@ -705,14 +705,12 @@ Note: The next minor version will focus on bringing more documentation for all t
 
 ### Important Notes
 
-* Some keys for various templates have changed to provide more flexibility and enable some new features in the templates. As a result, **if you created your own custom templates, they might not all be totally compatible with SwiftGen 3.0.0** (hence the new major version).
+- Some keys for various templates have changed to provide more flexibility and enable some new features in the templates. As a result, **if you created your own custom templates, they might not all be totally compatible with SwiftGen 3.0.0** (hence the new major version).
```

Comment:
Why the sudden change to `-` instead of `*`?

Reply:
Because those are notes, not CHANGELOG entries.

To be fair, those can be changed back now, that was at a time when my logic conditions in my check considered it an error to have changelog entries without links under them, but since those two are notes, not changelog entry, I used `-` instead of `*` here to not consider them entries but just a list unrelated to entries with links and all, so my RegEx would ignore them/not consider them as entries.

Now that I allow entries without links, because there were some in our current CHANGELOG and they were justified, I guess we could switch those two back to `*`… or not